### PR TITLE
Gateway: symmetric encryption

### PR DESCRIPTION
**Pull request by IceKhan13**
### Summary

Adds symmetric encryption for tokens while program is in queue

### CODE REVIEW

File: charts/quantum-serverless/charts/gateway/templates/deployment.yaml [hunk 1]
```diff
@@ -65,6 +65,8 @@ spec:
           env:
             - name: DEBUG
               value: {{ .Values.application.debug | quote }}
+            - name: DJANGO_SECRET_KEY
```
Comment (Tansito) [inline charts/quantum-serverless/charts/gateway/templates/deployment.yaml:68]: I think we can consider that this value will be usually taken from a secret store as the [`DATABASE_PASSWORD`](https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/charts/quantum-serverless/charts/gateway/templates/deployment.yaml#L118). I would change to something like:

```
            - name: DJANGO_SECRET_KEY
              valueFrom:
                secretKeyRef:
                  name: django
                  key: secret_key
```

what do you think?

File: charts/quantum-serverless/charts/gateway/values.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ useCertManager: false
 application:
 #  command: [ "gunicorn", "gateway.wsgi:application", "--bind", "0.0.0.0:8000", "--workers=3" ]
   debug: 0
+  secretKey: "django-insecure-&)i3b5aue*#-i6k9i-03qm(d!0h&662lbhj12on_*gimn3x8p7"
```
Comment (Tansito) [inline charts/quantum-serverless/charts/gateway/values.yaml:11]: If my previous comment apply we could remove this line letting only the default value in `gateway`:

https://github.com/Qiskit-Extensions/quantum-serverless/pull/905/files#diff-d3fab2ab912c6eb5c879651a1476f64f0551b747bf618b4ab13ddb4602dd45c1R26

**Review by Tansito — APPROVED** [review]
Thank you, @IceKhan13 ! In general I agree with implementation, just a minor comment that I think would be correct to add.

File: charts/quantum-serverless/charts/gateway/templates/deployment.yaml (hunk 1 above)
Comment (IceKhan13) [reply charts/quantum-serverless/charts/gateway/templates/deployment.yaml:68]: Could not agree more! Good catch!

**Review by Tansito — APPROVED** [review]
Nice! 🎉

**Pull request merged**